spec/models/maat_api/court_application_spec.rb

```diff
@@ -1,111 +1,123 @@
 RSpec.describe MaatApi::CourtApplication, type: :model do
-  let(:hearing_body) { JSON.parse(file_fixture("hearing/with_court_application.json").read).deep_symbolize_keys }
+  let(:shared_time) { "2021-03-09T15:54:09.871Z" }
```

Comment:
I don't think this is being used in this file

Reply:
Good spot, thanks!